tests/Makefile.am

```diff
@@ -78,6 +80,7 @@ AM_TESTS_ENVIRONMENT = \
 	$(if $(EOS_UPDATER_GDB),EOS_CHECK_UPDATER_GDB_BASH_PATH=$(EOS_UPDATER_GDB)) \
 	$(if $(EOS_UPDATE_SERVER_GDB_BASE),EOS_CHECK_UPDATE_SERVER_GDB_BASH_PATH_BASE=$(EOS_UPDATE_SERVER_GDB_BASE)) \
 	$(if $(EOS_UPDATER_PREPARE_VOLUME_GDB),EOS_CHECK_UPDATER_PREPARE_VOLUME_GDB_BASH_PATH=$(EOS_UPDATER_PREPARE_VOLUME_GDB)) \
+	TMPDIR=$(builddir)/tests-temp \
```

Comment:
Please generate a temp directory dynamically during the test. Using a hard-coded temp directory makes it a lot easier for stale state to persist between test runs, or for tests to collide if they’re run in parallel.

Reply:
Yep, I forgot about this part.

The reason why I had to override the temp prefix was because flatpak wants to set xattrs on the flatpak dirs which is not possible when they are on the tmpfs.

In this case, setting TMPDIR just sets the temp prefix, the individual directories created *should* still be randomised. It'd be nice if we could also remove it at the end of a successful test run, though I'm not sure how to do that with autotools.

Reply:
There’s got to be a better way of doing this. Could we not get flatpak to use a different OSTree repository mode (which doesn’t require xattrs) for the tests? Or is it settings xattrs on directories which aren’t part of the OSTree repo? Is there a flatpak option to not use xattrs?

Reply:
I'll have to do a deeper dive to figure that out, but yeah, if there's a way of doing this so that we can still run on the tmpfs, that'd be nice for sure.

Reply:
Yup, this was easily fixable by setting the `no-xattrs` flat on `OSTREE_SYSROOT_DEBUG`